mao-images.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+pod=$(oc get pod -n openshift-machine-api | grep '^machine-api-operator-')
```

Comment:
This will return the whole status line, I think you want something like `oc get pod -n openshift-machine-api -o name | grep '^pod/machine-api-operator-'`